rules/sonic-ztp.dep

```diff
@@ -0,0 +1,13 @@
+
+SPATH       := $($(SONIC_ZTP)_SRC_PATH)
+DEP_FILES   := $(SONIC_COMMON_FILES_LIST) rules/sonic-ztp.mk rules/sonic-ztp.dep   
+DEP_FILES   += $(SONIC_COMMON_BASE_FILES_LIST)
+SMDEP_FILES := $(addprefix $(SPATH)/,$(shell cd $(SPATH) && git ls-files|grep -Ev "inband-ztp-ip|dhclient-exit-hooks.d/ztp"))
```

Comment:
why do we need grep here?

Reply:
Fixed and uploaded the patchset.

Reply:
i can still see the grep, can you double check?